packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -272,13 +346,18 @@ The following parameters can be posted as a part of the URL-encoded form values
 | redirect_uri            | Required if `grant_type` is `authorization_code`. Specifies the callback location where the authorization was sent. This value must match the `redirect_uri` used to generate the original `authorization_code`.                                                                                                                   | String |
 | refresh_token           | Required if `grant_type` is `refresh_token`. The value is a valid refresh token that was returned from this endpoint previously.                                                                                                                                                                                                   | String |
 | scope                   | Required if `password` is the `grant_type`. This is a list of scopes that the client wants to be included in the access token. For the `refresh_token` grant type, these scopes have to be a subset of the scopes used to generate the refresh token in the first place.                                                             | String |
-| username                | Required if the grant_type is `password`.                                                                                                                                                                                                                                                                                          | String |
-| assertion               | Required if the grant_type is `urn:ietf:params:oauth:grant-type:saml2-bearer`<ApiLifecycle access="ea" />. | String |
+| username                | Required if the `grant_type` is `password`.                                                                                                                                                                                                                                                                                          | String |
+| assertion               | Required if the `grant_type` is `urn:ietf:params:oauth:grant-type:saml2-bearer`<ApiLifecycle access="ea" />. | String |
+| audience                | Required if the `grant_type` is `urn:ietf:params:oauth:grant-type:token-exchange`<ApiLifecycle access="ea" />. | String |
+| subject_token_type      | Required if the `grant_type` is `urn:ietf:params:oauth:grant-type:token-exchange`<ApiLifecycle access="ea" />. | String |
+| subject_token           | Required if the `grant_type` is `urn:ietf:params:oauth:grant-type:token-exchange`<ApiLifecycle access="ea" />. | String |
+| actor_token_type        | Required if the `grant_type` is `urn:ietf:params:oauth:grant-type:token-exchange`<ApiLifecycle access="ea" />. | String |
+| actor_token             | Required if the `grant_type` is `urn:ietf:params:oauth:grant-type:token-exchange`<ApiLifecycle access="ea" />. | String |
 
 #### Response properties
```

Comment:
it should also include device_secret. Do we want to include example for token exchange?

Reply:
The particular example of a token exchange call will be shown in the guide, so don't think it's appropriate to include it in this overall api reference about OAuth2/OIDC.